Doc/library/decimal.rst

```diff
@@ -23,6 +23,11 @@
    # make sure each group gets a fresh context
    setcontext(Context())
 
+.. testcleanup:: *
+
+   # make sure other tests (also outside this file) get a fresh context
+   setcontext(Context())
+
```

Comment:
Please leave decimal.rst changes out of this PR and open a separate PR and then ping Stefan Krah.